Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -983,13 +983,16 @@ open class NavigationMapView: UIView {
     
     /**
      Allows to control current user location styling based on accuracy authorization permission on iOS 14 and above.
+     Defaults to `true`.
      
-     If `false`, user location will be drawn based on style, which was set in `NavigationMapView.userLocationStyle`.
+     When user disable the `Precise Location` property in the settings of current application:
      If `true`, `UserHaloCourseView` will be shown.
+     If `false`, user location will be drawn based on style, which was set in `NavigationMapView.userLocationStyle`.
      */
-    @objc dynamic public var reducedAccuracyActivatedMode: Bool = false {
+    @objc dynamic public var reducedAccuracyActivatedMode: Bool = true {
```

Comment:
Changing the default value of a public property feels kind of a breaking change.

Reply:
Yes, I agree. Because previously when we set the `reducedAccuracyActivatedMode ` to `true`, it will automatically switch to the `UserHaloView` even the `precise location` is enabled. But it defaults to `false`, so when the  `precise location` is disabled, it will still show the `UserHaloView`, which causes some issue as in #4027 .

So I think this property should control whether we want to display the `UserHaloView` when the `precise location` is disabled. Because usually when `precise location` is enabled, we don't want to display this reduced view. If users don't want this automatic switch, they could change the It changes `reducedAccuracyActivatedMode ` to false. So even when the the `precise location` is disabled, it will still use the style defined `userLocationStyle`. 

I think it makes more sense but It changes the previous logic as well as the default value. So I add the ChangeLog entry for this change and also update the snapshot tests for these different circumstances.

Reply:
I updated the default value to `false` as before.